koku/koku/celery.py

```diff
@@ -175,8 +175,10 @@ def validate_cron_expression(expresssion):
 }
 
 # Specify the frequency for pushing source status.
-SOURCE_STATUS_FREQUENCY_MINUTES = ENVIRONMENT.get_value("SOURCE_STATUS_FREQUENCY_MINUTES", default="30")
-source_status_schedule = crontab(minute=f"*/{SOURCE_STATUS_FREQUENCY_MINUTES}")
+status_expression = "0 3 * * *"
+SOURCE_STATUS_SCHEDULE = ENVIRONMENT.get_value("SOURCE_STATUS_SCHEDULE", default=status_expression)
+SOURCE_STATUS_SCHEDULE = validate_cron_expression(SOURCE_STATUS_SCHEDULE, default=status_expression)
```

Comment:
Readability could be improved here by only assignming `SOURCE_STATUS_SCHEDULE` a validated value. There is also the possibility of an invalid value being passed to `crontab` since the fallback value passed to `validate_cron_expression` is not validated.

A cleaner flow would be:
1. Get the value from the environment or use a local default
2. Validate that value and assign it to `SOURCE_STATUS_SCHEDULE`

```suggestion
fallback = "0 3 * * *"
status_expression = ENVIRONMENT.get_value("SOURCE_STATUS_SCHEDULE", default=fallback)
SOURCE_STATUS_SCHEDULE = validate_cron_expression(status_expression, fallback)
```